src/sentry/static/sentry/app/components/tagDistributionMeter/index.jsx

```diff
@@ -61,7 +61,7 @@ export default class TagDistributionMeter extends React.Component {
 
           const tooltipHtml = (
             <React.Fragment>
-              <div className="truncate">{value.name}</div>
+              <div className="truncate">{value.name || t('n/a')}</div>
```

Comment:
@markstory The proptypes for `segments.name` and `segments.value` will need to be updated to have `isRequired` removed.

Reply:
Done.